ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -62,14 +62,14 @@ jobs:
         timeout: 1800
         topology: *build
 
-  fedora-27/temp_commit:
+  fedora-27/temp_trust:
     requires: [fedora-27/build]
     priority: 50
     job:
-      class: RunPytest
+      class: RunADTests
       args:
         build_url: '{fedora-27/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
+        test_suite: test_integration/test_trust.py
         template: *ci-master-f27
-        timeout: 3600
-        topology: *master_1repl_1client
+        timeout: 7200
+        topology: *ad_master_2client
```

Comment:
The topology needs to be `adroot_adchild_adtree_master_1client` for test_trust.

Reply:
When I use `adroot_adchild_adtree_master_1client`. tests fails for 
https://github.com/freeipa/freeipa/blob/f3e117156436bbf715232bea730a9172c14a8c1f/ipatests/test_integration/test_trust.py#L19